staynomad/Nomad-Back
# Explore listings

# Relevant issue

Closes #545

# Summary of change
- Fixed the "queryObj" to store the right query in the listing.controller.js file so that the query would work in listings.find(queryObj,...)
- There is another bug that I wasn't able to fix which was when you create a new listing on the frontend the backend somehow converts the "maxpeople" property into a string value instead of an integer.
# Testing/Verification

- Make sure to switch to the "Explore_listings" branch on Nomad Frontend to perform the following tests as well.
- Check if the family size and budget listings on the explore page correspond to the req parameters sent from the frontend (see photo)
- Try creating a new listing on the frontend and see if the listings show up in the appropriate listing slots (e.g Family Size should show 5 or more guests listings, and Budget should show houses starting from $100 and up for listings).
![image](https://user-images.githubusercontent.com/76442800/147508003-478074bc-9f66-4edc-bd0d-eca3d76cb222.png)
